GitCommands/Git/GitModule.cs

```diff
@@ -513,14 +513,16 @@ public bool EditNotes(ObjectId commitId)
             return _gitExecutable.RunCommand(arguments, createWindow: createWindow);
         }
 
-        public bool InTheMiddleOfConflictedMerge()
+        public bool InTheMiddleOfConflictedMerge(bool background = false)
```

Comment:
```suggestion
        public bool InTheMiddleOfConflictedMerge(bool throwErrors = true)
```

Reply:
If it should be aligned to the GitProcess: Renaming to `throwOnErrorExit`

Reply:
I don't think this is necessary. `throwOnErrorExit` has meaning within `GitProcess` implementation, i.e. throw an exception if the process faulted. However in the context of `InTheMiddleOfConflictedMerge` (and especially its callers) there is clear link to the underlaying process. Reading this code I get multiple questions `hasConflicts = Module.InTheMiddleOfConflictedMerge(throwOnErrorExit: false);` - e.g., "exit" of what? Hence I proposed `throwErrors`.

Reply:
I find `throwErrors` in isolation be more confusing, for what occasion are errors thrown?
`throwOnErrorExit` has information that this is limited to "something".

`throwOnErrorExit: false` is only set in one "background" operation right now, hopefully we can keep it at that.
This is the type of operations where an external operation (WSL command, not even Git) returns an error that GE can handle.
Maybe the user should be notified but not at every occurrence.
The solution in this case is for Microsoft to fix the WSL implementation.